### PR TITLE
Es7 support

### PR DESCRIPTION
**Pull request by zzacharo**

### CODE REVIEW

File: invenio_oaiserver/percolator.py [hunk 1]
```diff
@@ -75,15 +81,18 @@ def _get_percolator_doc_type(index):
...
-        query = query_string_parser(search_pattern=search_pattern).to_dict()
+        query = query_string_parser(
+            search_pattern=search_pattern,
+            search_fields=search_fields).to_dict()
         for index in current_search.mappings.keys():
             # Create the percolator doc_type in the existing index for >= ES5
             # TODO: Consider doing this only once in app initialization
             percolator_doc_type = _get_percolator_doc_type(index)
             _create_percolator_mapping(index, percolator_doc_type)
+            import ipdb; ipdb.set_trace()
```
Comment (lnielsen) [inline invenio_oaiserver/percolator.py]: pytest :-)

Comment (zzacharo) [reply invenio_oaiserver/percolator.py]: ah no it's a draft :) But thanks for the observation ;)

File: .travis-lowest-requirements.txt [hunk 2]
```diff
@@ -0,0 +1,26 @@
+arrow==0.13.0
```
Comment (slint) [inline .travis-lowest-requirements.txt]: You have a Travis leak

File: .travis.yml [hunk 3]
```diff
@@ -26,33 +26,28 @@ cache:
...
-    - ES6_DOWNLOAD_URL="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-6.2.2.tar.gz"
+    - ES5_DOWNLOAD_URL="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.6.16.tar.gz"
+    - ES6_DOWNLOAD_URL="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-6.8.1.tar.gz"
+    - ES7_DOWNLOAD_URL="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.2.0-linux-x86_64.tar.gz"
     - ES_HOST=127.0.0.1
   matrix:
-    # ES2 + SQLite
-    - REQUIREMENTS=lowest EXTRAS=all,sqlite,elasticsearch2 SQLALCHEMY_DATABASE_URI="sqlite:///test.db" ES_URL=$ES2_DOWNLOAD_URL
     # ES5 + MySQL
     - REQUIREMENTS=lowest EXTRAS=all,mysql,elasticsearch5 SQLALCHEMY_DATABASE_URI="mysql+pymysql://travis@localhost:3306/invenio" ES_URL=$ES5_DOWNLOAD_URL
```
Comment (slint) [inline .travis.yml]: We need another `lowest` build with ES6 (to have two-of-all)

Comment (zzacharo) [reply .travis.yml]: ok!

**Pull request merged**